src/sentry/tagstore/v2/backend.py

```diff
@@ -583,8 +583,8 @@ def get_top_group_tag_values(self, project_id, group_id, environment_id, key, li
                            tagstore_grouptagvalue.project_id,
                            tagstore_grouptagvalue.group_id,
                            tagstore_grouptagvalue.times_seen,
-                           tagstore_grouptagvalue.key_id,
-                           tagstore_grouptagvalue.value_id,
+                           tagstore_grouptagvalue.key_id as _key_id,
```

Comment:
this is what flushed out the weird .values() behavior, as it was binding as 'key_id' instead of '_key_id' which was effectively wrong in 1.8